docker-compose.ci.yml

```diff
@@ -24,12 +25,21 @@ services:
       - mysql_vol:/var/lib/mysql_tmpfs/
       - .:/obs
     command: /obs/contrib/start_test_db
-  backend:
+  tbackend:
     image: registry.opensuse.org/obs/server/unstable/container/sle12/sp3/images/openbuildservice/backend:latest
     volumes:
       - .:/obs
     working_dir: /obs
     command: make -C src/backend test
+  backend:
+    image: openbuildservice/backend
```

Comment:
Why not also running 
```
registry.opensuse.org/obs/server/unstable/container/sle12/sp3/images/openbuildservice/backend:latest
```
?

Reply:
I just copied from the dev container. It was a mere experiment